platform/Arg.roc

```diff
@@ -10,8 +10,9 @@ import Arg.ErrorFormatter exposing [formatArgExtractErr]
 import Arg.Help exposing [helpText, usageHelp]
 
 ## Gives a list of the program's command-line arguments.
-list : Task (List Str) []_
-list =
+## `{} ->` is a necessary workaround to make error accumulation/unification work.
+list : {} -> Task (List Str) []
+list = \_ ->
```

Comment:
For the old version the `[]_` would be inferred as `[Exit I32 Str, StdoutErr Stdout.Err]` because of its use in the `parse` function. This would require users to handle errors when using `list` that can not actually occur, which is also very confusing.